jujugui/static/gui/src/app/app.js

```diff
@@ -639,6 +639,47 @@ YUI.add('juju-gui', function(Y) {
     },
 
     /**
+      Sends the discharge token via POST to the storefront. This is used
+      when the GUI is operating in GISF mode, allowing a shared login between
+      the GUI and the storefront.
+
+      @method _sendGISFPostBack
+    */
+    _sendGISFPostBack: function() {
+      const dischargeToken = window.localStorage.getItem('discharge-token');
+      if (!dischargeToken) {
+        console.error('no discharge token in local storage after login');
+      } else {
+        console.log('sending discharge token to storefront');
+        const content = 'discharge-token=' + dischargeToken;
+        const webhandler = new Y.juju.environments.web.WebHandler();
```

Comment:
I know this is just code being moved around, but it would be nice to see a future branch move things from `Y.juju` over to `window.juju` as part of decommissioning YUI.